docs/framework/react/reference/useInfiniteQuery.md

```diff
@@ -66,19 +66,24 @@ The returned properties for `useInfiniteQuery` are identical to the [`useQuery`
   - Will be `true` while fetching the previous page with `fetchPreviousPage`.
 - `fetchNextPage: (options?: FetchNextPageOptions) => Promise<UseInfiniteQueryResult>`
   - This function allows you to fetch the next "page" of results.
-    `getNextPageParam`.
```

Comment:
I removed this line as it seemed unintentional/lacked context:

<img width="838" alt="image" src="https://github.com/TanStack/query/assets/13686180/af141b2d-a554-43d7-a850-5812022edf73">